app/request_handler.py

```diff
@@ -92,6 +93,14 @@ def readCachedVenueIdentifiers(cache):
         return cache.get("identifiers", None)
     return None
 
+def researchPlace(keyID, sourcesList, identifiers):
```

Comment:
nit: it's unclear that this function calls functions that write to the DB - can you rename it to make it clearer?